src/server/browser.ts

```diff
@@ -35,6 +36,7 @@ export interface BrowserProcess {
 export type PlaywrightOptions = {
   registry: registry.Registry,
   rootSdkObject: SdkObject,
+  portForwardingServer: BrowserServerPortForwardingServer,
```

Comment:
I don't like port forwarding server here. What do we need from it, just a port?

Reply:
just the forwarded Ports and to get the proxy server. Can also reduce it so that we have just these two values:
- forwardPorts
- proxyOverride